.github/CODEOWNERS

```diff
@@ -137,4 +137,5 @@ ui/components/ui/deprecated-networks                  @MetaMask/metamask-assets
 ui/components/ui/nft-collection-image                 @MetaMask/metamask-assets
 
 # Extension Platform
-yarnrc.yml                                            @MetaMask/extension-platform
\ No newline at end of file
+yarnrc.yml                                            @MetaMask/extension-platform
+mock-e2e-whitelisted.js                               @MetaMask/extension-platform
```

Comment:
I think this should be `test/e2e/mock-e2e-whitelisted.js`

Reply:
awwww  :crying_cat_face: thank you!!!